cmd/clusterctl/examples/google/machine_setup_configs.yaml

```diff
@@ -28,30 +28,32 @@ items:
           chmod $3 $2
       }
 
-      curl -sf https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
-      touch /etc/apt/sources.list.d/kubernetes.list
-      sh -c 'echo "deb http://apt.kubernetes.io/ kubernetes-xenial main" > /etc/apt/sources.list.d/kubernetes.list'
       apt-get update -y
-      apt-get install -y \
-        socat \
-        ebtables \
-        apt-transport-https \
-        cloud-utils \
-        prips
+      apt-get install -y socat ebtables apt-get install apt-transport-https ca-certificates curl software-properties-common cloud-utils prips
```

Comment:
Missing newline I think?

Reply:
Apparently `apt-get install` doesn't know how to install `apt-get` and `install`, fixed up in the latest version :joy: